src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public int getRandomNumber() {
+        return new Random().nextInt(100);
+    }
+
+    public String getRandomColor() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        return colorSupplier.getRandomColor();
+    }
+
+    public void getRandomBall() {
+        Ball ball = new Ball(getRandomColor(), getRandomNumber());
+        System.out.println(ball.toString());
+    }
```

Comment:
```suggestion
public class Lottery {
        ColorSupplier colorSupplier = new ColorSupplier();
        Random random = new Random();
        
    public int getRandomNumber() {
        return random.nextInt(100);
    }

    public String getRandomColor() {
        return colorSupplier.getRandomColor();
    }

    public void getRandomBall() {
        Ball ball = new Ball(getRandomColor(), getRandomNumber());
        System.out.println(ball);
    }
```

давай вынесем переменные Random и ColorSupplier на уровень класса, что б не создавать каждый раз новый экземпляр, когда мы вызываем метод getRandomBall()